packages/pluggableWidgets/slider-web/src/Slider.tsx

```diff
@@ -0,0 +1,38 @@
+import { createElement } from "react";
+import { SliderContainerProps } from "../typings/SliderProps";
+import { getMinValue } from "./utils/getMinValue";
+import { getMaxValue } from "./utils/getMaxValue";
+import { getStepValue } from "./utils/getStepValue";
+import { useMarks } from "./utils/useMarks";
+import { createHandleGenerator } from "./utils/createHandleGenerator";
+import { useOnChangeDebounced } from "./utils/useOnChangeDebounced";
+import { Slider as SliderComponent } from "./components/Slider";
+import "rc-slider/assets/index.css";
+import "rc-tooltip/assets/bootstrap.css";
+import "./ui/Slider.scss";
+import { isVertical } from "./utils/isVertical";
+
+export default function Slider(props: SliderContainerProps) {
+    const { valueAttribute, class: className, style } = props;
+    const minValue = getMinValue(props);
+    const maxValue = getMaxValue(props);
+    const stepValue = getStepValue(props);
+    const marks = useMarks(props);
+    const handle = createHandleGenerator(props);
+    const { onChange } = useOnChangeDebounced(props);
+
+    return (
+        <SliderComponent
```

Comment:
Normally we only use the first entry file to map widget properties into primitives (like DynamicValue -> number here) and then use `src/components/...tsx` to do additional UI stuff (in this case that would prob be the `useMarks`, `createHandleGenerator` and `useOnChangDebounced`. But for now, I think it's fine to leave it like this.

Reply:
It was such a small component, i also said to Illia to just use the same as you will need to handle the prop types in the entry points anyways.

Reply:
Yup that's fine. Just wanted to let him know, but for this widget it's fine cuz it's small